.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,58 @@
+|**Slot number**|<!-- Put the slot number you are using when the bug appears, from 1 to 8. If it is not applicable, put N/A. -->|
+|**Slot configuration**|<!-- Write the setting name of the slot you are using when the bug appears. The setting name might be one like "MF_CLASSIC_1K" or "MF_DETECTION", amongst the list found on wiki here: https://github.com/iceman1001/ChameleonMini-rebooted/wiki/Configurations. If it is not applicable, put N/A. -->|
+|**Dump source**|<!-- If you are using a card dump on the slot when the bug appears, briefly describe where your dump come from (card source, how did you dump it). If it is not applicable, put N/A. -->|
+|**Reader**|<!-- Briefly describe the reader or device to which you applied your ChameleonMini (brand, type, use case). If it is not applicable, put N/A. -->|
+|**Flashing environment**|<!-- Briefly describe the environment from which you compile, flash and program your CHameleonMini (OS, version). If you use different ones from different activities, or could reproduce the bug on different environments, describe them all. If it is not applicable, put N/A. -->|
+|**Flashing interface**|<!-- How do you connect your ChameleonMini to your computer to flash it. Please specify amongst: USB, AVRISPmkII, other (describe). If it is not applicable, put N/A. -->|
+|**Flashing method**|<!-- How do you flash a firmware to your ChameleonMini. Please specify amongst: dfu-programmer, flash.bat, BOOT_LOADER_EXE.exe, avrdude, other (describe). If it is not applicable, put N/A. -->|
+|**Flash memory space**|<!-- How many flash memory space do you have in your device. Find this information with "SPI_FLASHINFO" command. -->|
+|**Makefile configuration**|<!-- Tell which configuration/settings you tuned in Makefile. If it is not applicable, put N/A. -->|
+### About my use
+<!-- Please briefly describe how/why you mainly use a ChameleonMini, and what you are trying to do with it. -->
+### About me
+<!-- Put an "X" for each item you achieved or agree with, between [ ], like this: [X]. If you do not know, do not check. -->
+* [ ] I have knowledge on RFID/NFC standards and/or technologies
+* [ ] I know how to program (<!-- Please detail relevant langages that are used within this repo if you want: C, Bash, Python, etc. -->)
+* [ ] I have knowledge on AVR microcontrollers programming
+* [ ] I know how to do a Pull Request (PR), or am OK to learn it and try it
+
+## Bug description
+### Situation
+<!-- Please describe what you are trying to do when the bug appears, and in which situation. Please also describe the 2 or 3 last things you did with your ChameleonMini before you were in the situation where the bug appeared, if possible. -->
+### Expected function and references
+<!-- Please describe how things are supposed to work from your perspective, and give URL or paths to this repository or Wiki pages that actually tell this is how it is supposed to happen. -->
```

Comment:
Yet again fun,  but no.

Reply:
What's fun and no?

Reply:
the whole paragraph "expected function and references"